<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>449</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>581</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ShaderPaper</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>8</number>
    </property>
    <property name="topMargin">
     <number>8</number>
    </property>
    <property name="rightMargin">
     <number>8</number>
    </property>
    <property name="bottomMargin">
     <number>8</number>
    </property>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>23</height>
              </size>
             </property>
             <property name="text">
              <string>Update Rate</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="updateRateLabel">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>23</height>
              </size>
             </property>
             <property name="text">
              <string>--</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QSlider" name="updateRateSlider">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>120</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="pageStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>60</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="tickInterval">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>23</height>
              </size>
             </property>
             <property name="text">
              <string>Time Scale</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="timescaleLabel">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>23</height>
              </size>
             </property>
             <property name="text">
              <string>--</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QSlider" name="timescaleSlider">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="pageStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="tickInterval">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="mouseCheckBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Enable Mouse Input</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="forceButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Force Desktop Parent</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLineEdit" name="resWidth">
             <property name="text">
              <string>1920</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="resHeight">
             <property name="text">
              <string>1080</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="setResButton">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>23</height>
              </size>
             </property>
             <property name="text">
              <string>Apply</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
      <zorder>timescaleLabel</zorder>
      <zorder></zorder>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="galleryContainerLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>timescaleSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>timescaleLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updateRateSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>updateRateLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
